MeNext/Layout/SongListView.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System.Diagnostics;
+
+using Xamarin.Forms;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace MeNext
+{
+	// Song is a POD class for song information.
+	// TODO: maybe move this out of this file? Create a more legitimate version? Use the implementation
+	// from menext/music-service?
+	public class Song
+	{
+		public Song(string name)
+		{
+			this.Name = name;
+		}
+
+		public Song(string name, string artist)
+		{
+			this.Name = name;
+			this.Artist = artist;
+		}
+
+		// should we provide getters on these?
```

Comment:
Don't write getters! It's a property. The getter is inherent. It's just `get;`, which uses a default implementation. If you ever want to override that implementation, use:
```
get
{
// do stuff
return something;
}
```